test/e2e/specs/search/search__jetpack-instant.ts

```diff
@@ -24,151 +24,151 @@ declare const browser: Browser;
-		siteId = testAccount.credentials.testSites?.primary.id as number;
-		restAPIClient = new RestAPIClient( testAccount.credentials );
-	} );
+		beforeAll( async () => {
+			page = await browser.newPage();
 
-	skipItIf( envVariables.TEST_ON_ATOMIC )(
-		'Create a new post using the REST API',
-		async function () {
-			await restAPIClient.createPost( siteId, {
-				title: searchString,
-				content: searchString,
-			} );
-		}
-	);
-
-	skipItIf( envVariables.TEST_ON_ATOMIC )(
-		'Wait for index to update using the REST API',
-		async function () {
-			await waitForIndexToUpdate( restAPIClient, siteId, searchString );
-		}
-	);
+			testAccount = new TestAccount( accountName );
+			siteId = testAccount.credentials.testSites?.primary.id as number;
+			restAPIClient = new RestAPIClient( testAccount.credentials );
+		} );
 
-	it( 'Navigate to site homepage', async function () {
-		const isPrivateAtomicSite =
-			envVariables.TEST_ON_ATOMIC && envVariables.ATOMIC_VARIATION === 'private';
+		it( 'Create a post with the search block', async function () {
+			const response = await restAPIClient.createPost( siteId, {
+				title: postWithSearchBlockTitle,
+				content: '<!-- wp:search {"label":"Search","buttonText":"Search"} /-->',
+			} );
 
-		if ( isPrivateAtomicSite ) {
-			// First, get a fresh cookie.
-			await testAccount.authenticate( page );
-		}
+			postWithSearchBlockUrl = response.URL;
+		} );
 
-		// We can't wait for the "load" event because WordAds mess with it.
-		// But, if Playwright outpaces the search JS load, you can get weird artifacts like the search term getting wiped out.
-		// So we need to wait for the search JS to load before we start interacting with the page.
-		const waitForSearchJsPromise = page.waitForResponse(
-			( response ) =>
-				response
-					.url()
-					.includes( 'jetpack-search/build/instant-search/jp-search.chunk-main-payload.js' ),
-			{ timeout: 30 * 1000 }
+		skipItIf( envVariables.TEST_ON_ATOMIC )(
+			'Create a new post to be searched using the REST API',
+			async function () {
+				await restAPIClient.createPost( siteId, {
+					title: searchString,
+					content: searchString,
+				} );
+			}
 		);
 
-		await page.goto( testAccount.getSiteURL( { protocol: true } ), {
-			timeout: 20 * 1000,
-			waitUntil: 'domcontentloaded',
-		} );
+		skipItIf( envVariables.TEST_ON_ATOMIC )(
+			'Wait for index to update using the REST API',
+			async function () {
+				await waitForIndexToUpdate( restAPIClient, siteId, searchString );
+			}
+		);
 
-		if ( isPrivateAtomicSite ) {
-			// On private Atomic sites, still have to click the blue Log In button to use your cookie.
-			await page.getByRole( 'link', { name: 'Log in' } ).click();
-			// Then let all the redirects settle.
-			await page.waitForURL( testAccount.getSiteURL( { protocol: true } ), {
+		it( 'Navigate to post with search block', async function () {
+			// We can't wait for the "load" event because WordAds mess with it.
```

Comment:
**suggestion:non-blocking** - what if we abort requests to WordAds, like what I did here? https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/jest-playwright-config/environment.ts#L423-L432

Reply:
Oh, that's a good idea! We should definitely go back and add that to this spec at some point. 👍

(As an FYI, that didn't change in this PR though actually, it just looks diffed because I added the `skipDescribeIf`)